### PR TITLE
Automatically determine highest version when replacing RUNTIME_VERSION

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: for-all-boosters.sh [hunk 1]
```diff
@@ -388,8 +416,8 @@ release() (
         local -r sbVersion=${BASH_REMATCH[1]}
         local -r versionInt=${BASH_REMATCH[2]}
         local -r newVersionInt=$((versionInt + 1))
-        local -r qualifier=${BASH_REMATCH[3]}
-        local -r snapshot=${BASH_REMATCH[4]}
+        local qualifier=${BASH_REMATCH[3]}
+        local snapshot=${BASH_REMATCH[4]}
```
Comment (metacosm) [inline for-all-boosters.sh]: why these changes?

Comment (geoand) [reply for-all-boosters.sh]: They are part of the second commit of the PR.
The script was failing without these changes.

Comment (geoand) [reply for-all-boosters.sh]: And also had failed with #66, but I didn't know what to do about that

**Comment by metacosm** [comment]
Thank you!

**Comment by geoand** [comment]
You're welcome!
